tests/unit/test_yamlparser.py

```diff
@@ -113,13 +113,20 @@ def test_expand_env():
     assert expand_env_var('$PATH-${AA}') != '$PATH-${AA}'
 
 
-def test_encoder_name_env_replace():
+@pytest.fixture
+def set_test_encoder_env_var():
+    os.environ['BE_TEST_NAME'] = 'hello123'
+    yield
+    del os.environ['BE_TEST_NAME']
+
+
+def test_encoder_name_env_replace(set_test_encoder_env_var):
     os.environ['BE_TEST_NAME'] = 'hello123'
```

Comment:
u can remove this here, but is okey

Reply:
Ah, let me fix it so it's more consistent.